enterprise/api/v1/views/enterprise_group.py

```diff
@@ -200,6 +208,10 @@ def assign_learners(self, request, group_uuid):
                     )
                     for ecu in ecus
                 ]
+                # i think we still need to send invitation emails to 'realized' customers as well 
+                for ent_customer_user in ent_customer_users:
+                    if budget_expiration is not None and catalog_uuid is not None:
```

Comment:
I would put this outside of the for loop because it applies to all learners in this case and we'd only have to check once

Reply:
removing these lines since we are sending the emails below to all memberships